homeassistant/components/websocket_api/http.py

```diff
@@ -275,6 +275,15 @@ def handle_hass_stop(event: Event) -> None:
                 if msg.type in (WSMsgType.CLOSE, WSMsgType.CLOSED, WSMsgType.CLOSING):
                     break
 
+                if msg.type == WSMsgType.BINARY:
+                    if len(msg.data) < 2:
```

Comment:
```suggestion
                    if len(msg.data) < 1:
```
An empty payload is a valid message. I use this to signal the end of an audio stream.

Reply:
Added support for this and explicitly test that this works.